core/input/generisdata_tech.prn

```diff
@@ -121,12 +121,13 @@ lifetime                30       30          45          45          45
 *lifetime              40          40          40          40
 
 +                 ccsinje
-tech_stat               2
-inco0                 220
```

Comment:
do you need to also adjsut it in generisdata_tech_SSP1.prn and generisdata_tech_SSP5.prn?

Reply:
yes, you are right, I forgot that!